WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeWebViewFragment.java

```diff
@@ -118,6 +121,17 @@ public void onResume() {
         setNavigationIcon(R.drawable.ic_close_white_24dp);
     }
 
+    private void clearCookies() {
```

Comment:
> Good call. I'll move it to when the WebView closes (like iOS does)

Sorry, didn't finish my thought there. If it's at the end, it will solve the async issue of the webview showing up before the cookies are cleared.